payroll-processor-functions/src/payroll-processor-functions/local.settings.json.sample

```diff
@@ -0,0 +1,11 @@
+{
+  "IsEncrypted": false,
+  "Values": {
+    "AzureTableStorage": "UseDevelopmentStorage=true"
+  },
+  "Host": {
+    "LocalHttpPort": 7071,
+    "CORS": "*",
+    "CORSCredentials": false
+  }
```

Comment:
I copied this from one of the templates that creating a new azure function, with the scaffolding, creates.

I haven't worked with functions much since v1.x, and so much of the config/APIs have changed.

I'm not sure if any of this needs to be explicitly specified.